syntax = "proto3";
import "google/protobuf/empty.proto";
message CompleteData{
  float wellness = 1;
  float pollution = 2;
  string datetimew = 3;
  string datetimep = 4;
}
service Terminal_service {
  rpc send_results(CompleteData) returns (google.protobuf.Empty) {}
}

